jax/_src/lru_cache.py

```diff
@@ -31,6 +32,10 @@
 logger = logging.getLogger(__name__)
 
 
+_CACHE_SUFFIX = "-cache.bin"
```

Comment:
Do we need a cache suffix? I kinda feel the cache files should still be just the cache key

EDIT: oh I see it's so you can easily select only the cache entry files. You could do something like make the atime files hidden, like `f".{key}{_ATIME_SUFFX}`, and then use `glob.glob` which respects hidden files. Or filter out atime files manually in the loop. But I think what you have is good.

Please remove the .bin though (see below comment)